src/use-swr.ts

```diff
@@ -196,20 +74,23 @@ export const useSWRHandler = <Data = any, Error = any>(
 
   // Get the current state that SWR should return.
   const cachedData = cache.get(key)
-  const data = isUndefined(cachedData) ? initialData : cachedData
+  const fallbackData = isUndefined(initialData)
+    ? fallbackValues[key]
```

Comment:
I feel it would be wired if it's used as option for a hook. consider following

```js
useSWR(key, fn, {
   fallbackValues: { [key]: fallback }
})
```
This would look pretty duplicated for usage of key.

I feel the fallback should be different for hook and SWRConfig:
 
```js
<SWRConfig value={{ fallbacks: { [k1]: v1, [k2]: v2 } }}>
   <Page/ >
</SWRConfig>
```

```js
useSWR(key, fn, { fallback })
```

Reply:
A use case is to provide some global mockup data:

```js
const mockupData = { '/api/user': ..., '/api/items': ... }

useSWR(key, fn, { fallbackValues: mockupData })
```

Or, we can rename `initialData` to `fallbackValue` but also keep `fallbackValues` :D